mgmt/rpc/overridable_txn_vars.cc

```diff
@@ -0,0 +1 @@
+#include "../../src/shared/overridable_txn_vars.cc"
```

Comment:
The mgmt code moved to src/mgmt, but this file shouldn't be necessary regardless.  Please remove.

Reply:
Oh I don't even remember including this.. will remove